sysbuild/CMakeLists.txt

```
@@ -140,6 +140,9 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
 
 # TODO: SB_CONFIG_SOC_SERIES_NRF54HX
 
+  if(SB_CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS)
+    set_config_bool(mcuboot CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS y)
+  endif()
```

```suggestion
  if(SB_CONFIG_MCUBOOT_FPROTECT_ALLOW_COMBINED_REGIONS)
    set_config_bool(mcuboot CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS y)
  else()
    set_config_bool(mcuboot CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS n)
  endif()
```
Move to line 336, just above `# Apply image compression support options`